### PR TITLE
docs(rename): Update naming + name of new dep's

### PR DESCRIPTION
**Pull request by fokusferit**
Update naming + name of new dep's

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,11 +41,11 @@
     "preact-compat": "^3.16.0",
     "requirejs": "^2.3.2",
     "showdown": "^1.6.4",
-    "styleguide-web-components": "git+ssh://git@github.com/wholesale-design-system/components.git#master",
+    "fabric-components": "git+ssh://git@github.com/wholesale-design-system/components.git#master",
```
Comment (tonysaad) [inline package.json]: That still the old URL, right ?

File: package.json [hunk 2]
```diff
@@ -41,11 +41,11 @@
...
     "text": "github:requirejs/text#latest",
     "through2": "^2.0.3",
     "uglify-js": "^2.6.3",
     "vinyl-fs": "^2.4.3",
-    "wholesale-styleguide-scss": "git+ssh://git@github.com/wholesale-design-system/scss.git"
+    "fabric-scss": "git+ssh://git@github.com/wholesale-design-system/scss.git"
```
Comment (tonysaad) [inline package.json]: Here too

**Comment by fokusferit** [comment]
@tonysaad it should now work. Looks like there was a aurelia "task" which had hard references to the old repositories...

**Comment by tonysaad** [comment]
👍

**Review by tonysaad — APPROVED** [review]

**Pull request merged**